module/Company/src/Company/Service/Company.php

```diff
@@ -136,24 +148,15 @@ public function insertCompanyByData($data,$files)
         $companyForm = $this->getCompanyForm();
         $companyForm->setData($data);
         if ($companyForm->isValid()) {
-            //echo "hoi";
-            //var_dump($files);
             $company = $this->insertCompany($data['languages']);
             $company->exchangeArray($data);
-            //echo "hoi";
-            //var_dump($files);
             foreach ($company->getTranslations() as $translation) {
                 $file = $files[$translation->getLanguage() . '_logo'];
-                //var_dump($file);
                 try {
                     $newPath = $this->getFileStorageService()->storeUploadedFile($file);
-                    echo $newPath;
                     $translation->setLogo($newPath);
                 } catch (\Exception $exception) {
```

Comment:
Don't we need to do anything with these exceptions? Like return an error message?